app/views/dashboard/_buyer.html.erb

```diff
@@ -1,15 +1,45 @@
-<h1 style="margin-top: 120px; text-align:center;">Purchased Properties</h1>
 <% props = Property.where(buyer_id: current_user.id) %>
 
+<h1 style="margin-top: 130px; text-align:center;">Purchased Properties</h1>
+
 <section class="property-grid grid">
-      <div class="container">
-        <div class="row">
-          <div class="col-sm-12 justify-content-between">
-          <% props.each do |property| %>
-            <%= link_to property, class: "link-a" do  %>
-              <%= render property %>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+  <div class="container">
+    <div class="row">
+      <% props.each do |property| %>
+        <% seller = User.find(property.user_id) %>
```

Comment:
Remove this line and move on to the controller.

Reply:
- Moved lines to controller

Reply:
- Done